DOM+js+html/ui/script.js

```diff
@@ -0,0 +1,579 @@
+    // console.log(ob3_1);
+    // console.log(editPhotoPost("51", {
+    //     photolink: 'update photolink',
+    //     description: 'update description',
+    //     hashtags: new Set(["#soocool", "#2010", "#2018"]),
+    //     likes: new Set(["Mike", "Phil"])
+    // }));
+    // console.log(getPhotoPost("51"));
+    // console.log(removePhotoPost("51"));
+    //
+    // console.log("\nedit photopost false");
+    // let ob3_2 = new Photopost("52", "description52", new Date("2018-03-14T16:20:00"), "Vasia", "link", new Set([]), new Set(["#cool", "#2018"]));
+    // console.log(addPhotoPost(ob3_2));
+    // console.log(editPhotoPost("52", {}));
+    // console.log(getPhotoPost("52"));
+    //
+    // console.log("\n\n\nremove photopost true");
+    // let ob4 = new Photopost("61", "description22", new Date("2014-03-14T16:20:00"), "Vasia", "link", new Set([]), new Set(["#cool", "#2018"]));
+    // console.log(addPhotoPost(ob4));
+    // console.log(getPhotoPosts(0, 10));
+    // console.log(removePhotoPost("61"));
+    // console.log(getPhotoPosts(0, 10));
+    //
+    // console.log("\nremove photopost false");
+    // console.log(removePhotoPost("1250"));
+    return {
+        addPhotoPost,
+        getPhotoPosts,
+        removePhotoPost,
+        editPhotoPost
+    }
+}());
+
+
+let currentUser = null;
+let setuser = "Vasia";
+let dom = (function () {
+    function setUser(username) {
+
+        let isLogged = (currentUser !== null);
+        if (!isLogged && username !== "" && username !== null) {
+            currentUser = username;
+            document.getElementById('addedit').style.display = "block";
+            document.getElementById('usericon').setAttribute('src', 'images/face.jpg');
+            document.getElementById('username').style.display = "inline-block";
+            document.getElementById('username').innerHTML = username;
+            document.getElementById('log').innerHTML = "Log out";
+            showPostsToHtml();
+            return 0;
+        }
+        else if(isLogged){
+            document.getElementById('addedit').style.display = "none";
+            document.getElementById('usericon').setAttribute('src', 'images/user.png');
+            document.getElementById('username').style.display = "none";
+            document.getElementById('log').innerHTML = "Sign in";
+            currentUser = null;
+            showPostsToHtml();
+            return 0;
+        }
+    }
+
+    function allHashtags(set){
+        let str = "";
+        set.forEach(function(value) {
+            str += value + " ";
+        });
+        return str;
+    }
+
+    function showPostsToHtml() {
+        let container = document.getElementById('column');
+
+        while (container.firstChild) {
+            container.removeChild(container.firstChild);
+        }
+        let htmlPhotoPosts = js.getPhotoPosts();
+        for (let i = 0; i < htmlPhotoPosts.length; i++) {
+            if(htmlPhotoPosts[i].isDeleted === false) {
```

Comment:
you need to check this condition in your js module (and you do it, am I right)? why do you check if post is deleted again?
and your ``if`` statement should look like `` if (!htmlPhotoPosts[i].isDeleted)``